### DISCUSSION TITLE
Looking to contribute

### DISCUSSION THREAD

**Discussion by Captain-Robertson** [Captain-Robertson/Stormworks---Bungeling-Raiders-Hostile-A.I.-#5] [General]
Anyone looking to help out with this please let me know and I'll give you access to make changes 'etc'.
Try to keep things on the experimental branch to begin with and if it works as intended I'll merge it into the main one.

Best,

**Comment by Haiwei1027** [comment]
Hi, I would like to contribute

**Reply by Captain-Robertson** [reply]
Excellent! 
Request sent, let me know if you manage to get it and if it works okay, as Shad0kin says his request link keeps coming up as being invalid.

**Comment by Captain-Robertson** [comment]
What's your username on steam Haiwei? I'll add you as a co-creator because your help so far has been invaluable, there's no way I could have resolved even half of these issues by myself.

**Reply by Haiwei1027** [reply]
Thank you, I think your idea is brilliant and happy I'm able to help. Here's my steam profile. https://steamcommunity.com/profiles/76561198168069686/
